proposals/fh_discord.md

```diff
@@ -0,0 +1,47 @@
+# Florida Hackers Discord Chat 
+
+## Target audience
+
+<!-- Who are you expecting to impact based on your proposal? -->
+Not only will the discord improve communication for FH Organizers with permission based rooms(The ability to create rooms made for specific eyes only based on roles assigned in discord, which an infinite amount of roles can be assigned to, where rooms can be seen by everyone or a select few depending on assigned role), it also allows a more open community of Florida Hackers to be active on discord from the rest of Florida.
+
+## Community needs
+
+<!-- What specific issue are you addressing with your proposal? -->
+When it comes to determing whether or not Discord should be used over Slack, it's important to note that Slack isn't entirely free and has a data limit for messages and files for the free tier(That info can be found here https://get.slack.help/hc/en-us/articles/115002422943-Message-and-storage-limits-on-the-Free-plan). 
```

Comment:
> Slack isn't entirely free

I'd remove this statement; Slack _is_ free but you don't get all of the bells & whistles that you would if you paid for it.